### PR TITLE
https://issues.redhat.com/browse/ACM-11956

### PR DESCRIPTION
**Pull request by dockerymick**

### CODE REVIEW

File: governance/config_policy_ctrl.adoc [hunk 1]
```diff
@@ -138,6 +138,10 @@ By default, the minimum time between evaluations for configuration policies is a
 | `spec.object-templates[].recordDiff`
 | Optional
 | Specifies if and where to log the difference between the object on the cluster and the `objectDefinition` in the policy. Set to `Log` to log the difference in the controller logs or `None` to not log the difference. By default, this parameter is empty to not log the difference.
+//what is "dry run update support"
```
Comment (mprahl) [inline governance/config_policy_ctrl.adoc]: Does this help?
https://kubernetes.io/docs/reference/using-api/api-concepts/#dry-run

Essentially, config-policy-controller performs a simulated (i.e. dry run) update of the object with the Kubernetes API to see what the API says would have changed if it was a real request to further validate if the object in fact needs updating.

Comment (dockerymick) [reply governance/config_policy_ctrl.adoc]: @mprahl thanks Matt! Yes, this does help

**Review by mprahl — APPROVED** [review]

**Review by swopebe — APPROVED** [review]

File: governance/config_policy_ctrl.adoc [hunk 2]
```diff
@@ -139,6 +139,10 @@ By default, the minimum time between evaluations for configuration policies is a
 | Optional
 | Specifies if and where to log the difference between the object on the cluster and the `objectDefinition` in the policy. Set to `Log` to log the difference in the controller logs or `None` to not log the difference. By default, this parameter is empty to not log the difference.
 
+| `spec.object-templates[].recreateOption`
+| Optional
+| Describes whether to delete and recreate an object when an update is required. When set to `IfRequired` the policy recreates the object when you update an immutable field. When the parameter is set to `Always` the policy recreates the object if a mismatch is detected. The parameter value `RecreateOption` has no effect when the `remediationAction` is set to `inform`. The `IfRequired` value has no effect on clusters without dry-run update support. The default value is `None`.
```
Comment (jc-berger) [inline governance/config_policy_ctrl.adoc]: Optional: 

I read it "delete and recreate" as one action for the user to do, and I expected another action to follow. Consider this rephrase which also makes it more user focused: 

```suggestion
| Describes if you need to delete or recreate an object when an update is required. When you set the object to `IfRequired`, the policy recreates the object when you update an immutable field. When you set the parameter to `Always`, the policy recreates the object if a mismatch is detected. When you set the `remediationAction` to `inform`, the parameter value, `RecreateOption`, has no effect on your object. The `IfRequired` value has no effect on clusters without dry-run update support. The default value is `None`.
```

Comment (jc-berger) [reply governance/config_policy_ctrl.adoc]: I wasn't sure what the value had no effect on. The object? The cluster? Change my wording as needed.

Comment (dockerymick) [reply governance/config_policy_ctrl.adoc]: @jc-berger Thanks for your feedback Jake !

File: governance/config_policy_ctrl.adoc [hunk 3]
```diff
@@ -139,6 +139,10 @@ By default, the minimum time between evaluations for configuration policies is a
 | Optional
 | Specifies if and where to log the difference between the object on the cluster and the `objectDefinition` in the policy. Set to `Log` to log the difference in the controller logs or `None` to not log the difference. By default, this parameter is empty to not log the difference.
 
+| `spec.object-templates[].recreateOption`
+| Optional
+| Describes if you need to delete or recreate an object when an update is required. When you set the object to `IfRequired`, the policy recreates the object when you update an immutable field. When you set the parameter to `Always`, the policy recreates the object if a mismatch is detected. When you set the `remediationAction` to `inform`, the parameter value, `RecreateOption`, has no effect on your object. The `IfRequired` value has no effect on clusters without dry-run update support. The default value is `None`.
```
Comment (mprahl) [inline governance/config_policy_ctrl.adoc]:
```suggestion
| Describes when to delete and recreate an object when an update is required. When you set the object to `IfRequired`, the policy recreates the object when updating an immutable field. When you set the parameter to `Always`, the policy recreates the object on any update. When you set the `remediationAction` to `inform`, the parameter value, `recreateOption`, has no effect on the object. The `IfRequired` value has no effect on clusters without dry-run update support. The default value is `None`.
```

Comment (dockerymick) [reply governance/config_policy_ctrl.adoc]: @mprahl thanks Matt

**Pull request merged**